src/main/java/uk/gov/companieshouse/search/api/service/search/impl/alphabetical/AlphabeticalSearchRequestService.java

```diff
@@ -33,16 +33,26 @@
     private static final String ORDERED_ALPHA_KEY_WITH_ID = "ordered_alpha_key_with_id";
     private static final int FALLBACK_QUERY_LIMIT = 25;
 
+    @Override
+    public SearchResults getAlphabeticalSearchResults(String corporateName, String requestId) throws SearchException {
```

Comment:
What is this method used for?

Reply:
Good point, this was the original method which was overloaded to add the additional parameters. Amended to correct this.